### PR TITLE
Job detail layout

### PR DESCRIPTION
**Pull request by carlosxjm**

### CODE REVIEW

File: pages/job/_slug.vue [hunk 1]
```diff
@@ -39,8 +32,22 @@ export default {
...
+      } else if (this.job.maximumSalary) {
+        return `< R$ ${this.job.maximumSalary}`
+      }
+
+      return null
+    },
+  },
   mounted() {
     this.getJob()
+    console.log(this.$options)
```
Comment (allanhal) [inline pages/job/_slug.vue]: Needed?

File: pages/job/_slug.vue [hunk 2]
```diff
@@ -39,8 +32,22 @@ export default {
       job: Object,
     }
   },
+  computed: {
+    salaryRange() {
+      if (this.job.minimumSalary && this.job.maximumSalary) {
+        return `R$ ${this.job.minimumSalary} - R$ ${this.job.maximumSalary}`
```
Comment (allanhal) [inline pages/job/_slug.vue]: `return `R$ ${this.job.minimumSalary} ~ R$ ${this.job.maximumSalary}``

File: plugins/axios.js [hunk 3]
```diff
@@ -6,6 +6,8 @@ export default (ctx, inject) => {
   ctx.$axios.interceptors.request.use(
     (config) => {
       if (ctx.store.state.token) {
+        console.log(ctx.store.state.token)
```
Comment (allanhal) [inline plugins/axios.js]: Needed?

File: pages/job/_slug.vue (hunk 1 above)
Comment (carlosxjm) [reply pages/job/_slug.vue]: No

**Pull request merged**